services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/api/rest/_health.py

```diff
@@ -24,7 +24,7 @@ class HealthCheckError(RuntimeError):
     """Failed a health check"""
 
 
-@router.get("/", response_class=PlainTextResponse)
+@router.get("/health", response_class=PlainTextResponse)
```

Comment:
and typically this would be called `/live` I think.
see my comments above

Reply:
Actually the `/live` is breaking the convention, all the others are called `/health`